shared/js/background/utils.es6.js

```diff
@@ -193,7 +193,12 @@ function isBrokenList (url, lists) {
     // If root domain in temp unprotected list, return true
     return lists.some((brokenSiteDomain) => {
         if (brokenSiteDomain) {
-            return hostname.match(new RegExp(brokenSiteDomain + '$'))
+            // TODO: Remove string check after config migration
+            if (brokenSiteDomain instanceof String) {
+                return hostname.match(new RegExp(brokenSiteDomain + '$'))
+            } else {
+                return hostname.match(new RegExp(brokenSiteDomain.domain + '$'))
```

Comment:
is this correct? Isn't this going to match "testexample.com" to "example.com"?

Reply:
Yes. But that raises so many questions 👀